### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by StrongestNumber9**
This is on top of #55 so the diff will be enormous so don't look this before that is merged

### CODE REVIEW

File: zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/YarnInterpreterLauncherIntegrationTest.java [hunk 1]
```diff
@@ -79,28 +80,6 @@ public static void tearDown() throws IOException {
     }
   }
 
-  @Test
-  public void testLaunchShellInYarn() throws YarnException, InterpreterException, InterruptedException {
```
Comment (kortemik) [inline zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/YarnInterpreterLauncherIntegrationTest.java:83]: why this is removed?

File: zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java [hunk 2]
```diff
@@ -68,92 +67,15 @@ public static void setUp() throws Exception {
...
+    sparkHome = DownloadUtils.downloadSpark("3.4.2", "3");
   }
 
   @AfterClass
   public static void destroy() throws Exception {
     AbstractTestRestApi.shutDown();
   }
 
-  @Test
-  public void testZSession_Shell() throws Exception {
```
Comment (kortemik) [inline zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java:81]: why this is removed?

File: zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java [hunk 3]
```diff
@@ -68,92 +67,15 @@ public static void setUp() throws Exception {
...
-      assertEquals(4, note.getParagraphCount());
-      assertEquals("%sh invalid_command", note.getParagraph(3).getText());
-
-    } finally {
-      session.stop();
-    }
-  }
-
-  @Test
-  public void testZSession_Shell_Submit() throws Exception {
```
Comment (kortemik) [inline zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java:118]: why this is removed

**Review by kortemik — CHANGES_REQUESTED** [review]

File: zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/YarnInterpreterLauncherIntegrationTest.java (hunk 1 above)
Comment (StrongestNumber9) [reply zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/YarnInterpreterLauncherIntegrationTest.java:83]: Shell interpreter related test, will re-introduce and just `@Ignore` it

File: zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java (hunk 2 above)
Comment (StrongestNumber9) [reply zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java:81]: Shell interpreter related test, will re-introduce and just `@Ignore` it

File: zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java (hunk 3 above)
Comment (StrongestNumber9) [reply zeppelin-interpreter-integration/src/test/java/org/apache/zeppelin/integration/ZSessionIntegrationTest.java:118]: Shell interpreter related test, will re-introduce and just `@Ignore` it

**Review by kortemik — APPROVED** [review]

**Pull request merged**